library/Zend/InputFilter/InputFilterAbstractServiceFactory.php

```diff
@@ -52,9 +49,8 @@ public function canCreateServiceWithName(ServiceLocatorInterface $inputFilters,
      * @param string                  $rName
      * @return \Zend\InputFilter\InputFilterInterface
      */
-    public function createServiceWithName(ServiceLocatorInterface $inputFilters, $cName, $rName)
```

Comment:
so the method signature has to be changed to AbstractPluginManager ?
For me was the problem that the ServiceLocatorInterface doesnt have the method getServiceLocator.